tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -164,12 +175,27 @@ public async Task ConfidentialClientWithRSACertificateTestAsync()
         [TestMethod]
         public async Task ConfidentialClientWithClientSecretTestAsync()
         {
-            var confidentialClientAuthority = TestAuthority;
+            await RunTestWithClientSecretAsync(PublicClientConfidentialClientID,
+                                                           PublicCloudTestAuthority,
+                                                           _publicCloudCcaSecret).ConfigureAwait(false);
+        }
+
+        [TestMethod]
+        public async Task ArlingtonConfidentialClientWithClientSecretTestAsync()
```

Comment:
Would be good to test the `.WithCertificate` option as well. It's probably more imporant than `WithSecret`